test/integration/basic_test.go

```diff
@@ -40,6 +40,14 @@ type testCase struct {
 	desc string
 }
 
+func TimeBasedElectionId() p4_v1.Uint128 {
+	now := time.Now()
+	return p4_v1.Uint128{
+		High: uint64(now.Unix()),
+		Low:  uint64(now.UnixNano() % 1e9),
+	}
+}
```

Comment:
Could be imported from `pfcpiface` package, but I'm not sure of the implications.

Reply:
let's keep two separate instances

Reply:
fine with me. Two is still low and they have the same name.